frontend/src/data/constants.ts

```diff
@@ -46,13 +46,23 @@ export const OFFER_STATUS_OPTIONS = enumValues(OfferStatus).map((routeKey) => ({
   value: routeKey,
 }))
 
+/**
+ * Order of shipment statuses:
+ * draft → announced → open to offers → aid matching → staging at a hub
+ * → in transit → complete
```

Comment:
This is the result of a long discussion with Dom and Elliot 😄

Reply:
**thought:** we could express this as a state machine (e.g. using [xstate](https://xstate.js.org/) and validate state transitions (both in the UI and the backend) using that. This is only useful if there are actually restrictions on how the state can be change.